File: testdata/javascript/level-08-cross-file-calls/src/logger.js
export function info(msg) {
  console.log("[INFO] " + msg);
}

export function warn(msg) {
  console.log("[WARN] " + msg);
}

export class Logger {
  constructor(prefix) {
    this.prefix = prefix;
  }

  log(msg) {
    console.log(this.prefix + ": " + msg);
  }
}

File: testdata/rust/level-14-wildcard-imports/src/logger.rs
pub struct Logger {
    pub prefix: String,
}

impl Logger {
    pub fn new(prefix: &str) -> Logger {
        Logger { prefix: prefix.to_string() }
    }

    pub fn log(&self, msg: &str) {
        println!("{}: {}", self.prefix, msg);
    }
}
